ext/fetch/23_request.js

```diff
@@ -353,7 +354,15 @@
         request = cloneInnerRequest(originalReq, true);
         request.redirectCount = 0; // reset to 0 - cloneInnerRequest copies the value
         signal = input[_signal];
+        isRelative = originalReq.isRelative;
       }
+      // add `isRelative` as read-only property (instead of just doing
+      // `request.isRelative = isRelative;`) to prevent accidental or undesired changes
+      ObjectDefineProperty(request, "isRelative", {
+        value: isRelative,
+        writable: false,enumerable: false,
+        configurable: false
```

Comment:
Just a small style nitpick :) 
```suggestion
        writable: false,
        enumerable: false,
        configurable: false,
```

Reply:
Thx, that was a reformatting accident :)